prow/config.yaml

```diff
@@ -4711,6 +4749,53 @@ presubmits:
       - emptyDir: {}
         name: auto-generated-docker-graph-volume-mount
     trigger: (?m)^/test pull-security-kubernetes-local-e2e,?(\s+|$)
+  - agent: kubernetes
+    always_run: false
+    branches:
+    - master
+    cluster: security
+    context: pull-security-kubernetes-local-e2e-containerized
+    labels:
+      preset-k8s-ssh: "true"
+      preset-service-account: "true"
+    max_concurrency: 8
```

Comment:
Needs to be removed from here too I guess

Reply:
👍  doing it now

Reply:
looks like hack/update-config.sh defaults to `max_concurrency: 0`